test/cypress/integration/pages/obtener-respuestas.js

```diff
@@ -0,0 +1,28 @@
+import { ObtenerRespuestasBuscar } from '../../pages/obtener-respuestas/buscar';
+
+const buscar = new ObtenerRespuestasBuscar();
+
+describe( 'obtener-respuestas', () => {
+  describe( 'Buscar', () => {
+    it( 'should autocomplete results', () => {
+      buscar.open();
+      buscar.enter( 'divulgación de cierre' );
+      buscar.autocomplete().should( 'be.visible' );
+    } );
+
+    it( 'should return results', () => {
+      buscar.open();
+      buscar.enter( 'divulgación de cierre' );
+      buscar.search();
+      buscar.resultsSection().should( 'be.visible' );
+    } );
+
+    it( 'should correct spelling', () => {
+      buscar.open();
+      buscar.enter( 'vehíclo' );
+      buscar.search();
+      buscar.resultsHeader().contains( 'results for “vehículo”' );
+      buscar.resultsHeader().siblings( 'p' ).first().contains( 'Search instead for' );
```

Comment:
I think the text for these messages will be fully in spanish right? When checking this search on cf.gov the text appears as

![image](https://user-images.githubusercontent.com/16838497/101906220-9cf6ab80-3b86-11eb-9ba3-bfcb2498e4c5.png)

Reply:
Yes, I was going to comment on this as well, but got held up by my weird issues with cross-origin errors when running Cypress locally. Did you see my chat comment about that?

Reply:
Change english text to spanish in obtener-respuestas.js